medusa/providers/torrent/html/tvchaosuk.py

```diff
@@ -135,9 +135,11 @@ def parse(self, data, mode, **kwargs):
             # Skip column headers
             for row in torrent_rows[1:]:
                 try:
+                    # Skip highlighted torrents
+                    if (row['class'] == 'highlight even') or (row['class'] == 'highlight odd'):
```

Comment:
You don't need the brackets here.
Are you sure 'class' will always be there? If you are unsure and want to play it safe, use .get('class') instead.